docs/core/testing/unit-testing-mstest-runner-intro.md

```diff
@@ -139,9 +139,38 @@ All tests are executed until your test with a breakpoint is reached. Step throug
 
 ### Use Test Explorer
 
-`Microsoft.Testing.Platform` tests integrate with Visual Studio's **Test Explorer**. Tests can be run and debugged from **Test Explorer**.
+To run a test, navigate to **Test Explorer**, select the test (or tests) to run. Right select it, and choose **Run**. Similarly to debug a test, select the test (or tests) in **Test Explorer**, right select and choose **Debug**.
+
+> [!TIP]
```

Comment:
I don't think this is a tip, tip should make the user more successful, either NOTE or CAUTION

https://learn.microsoft.com/en-us/contribute/content/markdown-reference#alerts-note-tip-important-caution-warning

Reply:
Yes good point!

Reply:
I just copied verbatim what this header was for the VS tab in the documentation